src/styles/header.scss

```diff
@@ -0,0 +1,99 @@
+.header {
+    position: fixed;
+    top: 0;
+    left: 0;
+  
+    display: flex;
+    align-items: center;
+    justify-content: start;
+    flex-direction: column;
+  
+    height: 100%;
+    width: 160px;
+    padding-top: 20px;
+  
+    background-color: rgb(48, 48, 48);
+  
+    overflow-x: hidden;
+    z-index: 1;
+  }
+  
+  .container {
+    position: relative;
+    display: block;
+    margin: 10px 0;
+    transition: all .3s;
+    cursor: pointer;
+  
+    &:not(.triangle, .dotted-line) {
+      background: #ffffff;
+    }
+  
+    &:hover:not(.triangle, .dotted-line) {
+      background: peachpuff;
+    }
+  
+    &.rectangle {
+      width: 80px;
```

Comment:
If we set the width or height by JS, do it at the same place. We don't need conflicted styles and in this case is gonna be here. Let's remove it from .css. I think the shapes shouldn have any styles at all cause you generate them in JS. Set the styles there